docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,118 @@
+---
+title: GX Overview
+---
+
+import PythonVersion from '/docs/components/_python_version.mdx';
+
+Great Expectations is a framework for defining Expectations and running them against your data. Like assertions in traditional Python unit tests, Expectations provide a flexible, declarative language for describing expected behavior. Unlike traditional unit tests, GX applies Expectations to data instead of code. For example, you could define an Expectation that a column contains no null values, and GX would run that Expectation against your data, and report if a null value was found.
+
+## Before you start
+
+Before you've even installed GX, working with GX starts with data.  Specifically, data you care about enough for quality to be important.  We call your data, in its original format, your **Source Data**.  The combination of your data in its original format and the location at which it is stored is your **Source Data System**.
```

Comment:
```suggestion
GX was created to help ensure the quality and consistency of your data. The term given to data in its original format, is  **Source Data**.  The location where your source data is stored is defined as the **Source Data System**.
```
We need to stop mixing pronouns and get away from using _we_. The antecedent for the pronoun must be clear. I recommend avoiding _we_.

Reply:
Hey @Rachel-Reverie I think this is great initial work. It gets specific into a lot of the what to do's of GX and what different parts are. But as a first time user of GX, I feel like I'd be equally if not more curious about "Why GX"? Take a step back and clarifying the purpose of using GX - specific benefits and unique advantages of using GX of over frameworks - in the initial parts of the text would really hook the readers. 

I sort of lost focus after paragraph 5 since there was too much text my attention span isn't great with writing and is usually lengthened through visual guides.